css/properties/z-index.json

```diff
@@ -40,6 +40,38 @@
             "deprecated": false
           }
         },
+        "auto": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "1"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "1"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "1"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror"
```

Comment:
So for webview_android, this would produce:
z-index: ≤37
z-index.auto: 4.4

Wouldn't it be more correct for a scripted change to say ≤37 as well?

Reply:
Since we have increased the leniency of ranged versions, I think we should change the mirroring logic to say "<=4.4" instead, WDYT?

Reply:
> I think we should change the mirroring logic to say "≤4.4" instead of "≤37",

That's what the mirror script does today already. Right? 

Here the parent feature, z-index, is set to "≤37" manually and if you now set the child feature to mirror that produces "4.4". Data-validation wise, this is okay of course, but to me it is weird to have the parent left at "≤37" when we set a child to the more precise version of 4.4.

Reply:
Oh, my apologies, I misunderstood the problem!

"≤37" is a little less accurate than "≤4.4", so let me push a commit to set WebView to mirror on the other features!